include/mapbox/geojson_value_impl.hpp

```diff
@@ -0,0 +1,295 @@
+#pragma once
+
+#include <mapbox/geojson.hpp>
+#include <mapbox/geojson/value.hpp>
+
+namespace mapbox {
+namespace geojson {
+
+using error = std::runtime_error;
+
+template <typename T>
+T convert(const value &);
+
+template <>
+point convert<point>(const value &val) {
+    assert(val.is<value::array_type>());
+    const auto &valuePoint = *val.getArray();
+    if (valuePoint.size() < 2) {
+        throw error("coordinates array must have at least 2 numbers");
+    }
+
+    const auto getDouble = [](const auto &numVal) -> double {
+        return numVal.match([](double n) { return n; }, [](uint64_t n) { return n; },
```

Comment:
nit: move `[](uint64_t n) { return n; }` to next line

Reply:
That line was formatted by clang-format, I can revert that change.